api/src/test/java/org/openmrs/module/pihcore/TestAddressBundle.java

```diff
@@ -1,46 +1,45 @@
-package org.openmrs.module.pihcore.deploy.bundle.sierraLeone;
+package org.openmrs.module.pihcore;
```

Comment:
This is a whole new file, copied (and edited) from Haiti Core, not sure why it's showing up as a diff.